Contributing/UpdatingRequirements.md

```diff
@@ -0,0 +1,21 @@
+### Build Requirements
+
+The current list of build requirements is listed in `README.md`:
+- Xcode version
+- Deployment targets
+
+https://xcodereleases.com can be used as a reference.
+
+### Places to Update
+
+If these requirements change, there are several places that need to be updated:
+- `README.md`
+- https://www.revenuecat.com/docs/getting-started
```

Comment:
[This one](https://www.revenuecat.com/docs/ios#requirements) is currently wrong: 
![Screenshot 2023-01-18 at 08 01 45](https://user-images.githubusercontent.com/685609/213225461-9fb2db45-5be6-411b-ab8e-bd889d631ef5.png)

Reply:
Fixed by @aboedo, thanks!